vnext/ReactUWP/Views/DynamicAutomationPeer.cpp

```diff
@@ -200,6 +213,45 @@ void DynamicAutomationPeer::Select() const {
   DynamicAutomationProperties::DispatchAccessibilityAction(Owner(), L"select");
 }
 
+// IRangeValueProvider
+double DynamicAutomationPeer::Minimum() {
+  return GetAccessibilityValueRange(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Min);
+}
+
+double DynamicAutomationPeer::Maximum() {
+  return GetAccessibilityValueRange(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Max);
+}
+
+double DynamicAutomationPeer::Value() {
+  return GetAccessibilityValueRange(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Now);
+}
+
+// TODO
+// RN doesn't implement this
+double DynamicAutomationPeer::SmallChange() {
+  return 0;
+}
+
+// TODO
+// RN doesn't implement this
+double DynamicAutomationPeer::LargeChange() {
+  return 0;
+}
+
+// TODO 
+// RN doesn't implement this
+bool DynamicAutomationPeer::IsReadOnly() {
+  return false;
+}
+
+void DynamicAutomationPeer::SetValue(double value) {
+  DynamicAutomationProperties::DispatchAccessibilityAction(Owner(), L"setValue");
```

Comment:
is this enough? That's what eg. ISelectionItemProvider does for RemoveFromSelection